NOAA-EMC/godas
# Feature/ocn da diag table

## Description

- to generate ocean history files into files of ocn_da, which save fields for ocean DA in the native grids of MOM6

### Issue(s) addressed

- fixes #354 ( https://github.com/NOAA-EMC/godas/issues/354 )

## Testing

- Run with the diag_table and check the hourly files of ocn_da_yyyy_mm_dd_hh.nc
- Check the fields in the ocn_da files : geolon, geolat, SSH, MLD, Temp, Salt, u, v, h


## Dependencies

- Ocean DA process as model data
